Neos.ContentRepository.Core/Classes/SharedModel/Id/UuidFactory.php

```diff
@@ -30,4 +30,16 @@ public static function create(): string
             /** @phpstan-ignore-next-line not always true */
             : Uuid::uuid4()->toString();
     }
+
+    public static function createHex(): string
+    {
+        $uuid = self::create();
+
+        return str_replace('-', '', $uuid);
```

Comment:
Is that the actual HEX representation of a UUID v4? And is it the responsibility of the UUID factory to create hex and bin representations? (probably not an issue, I'm just thinking out loud)

Reply:
seemed easier to do it here, I will check but I think this should be correct, the dashes are just added for readability.